src/EA.Iws.Web/EA.Iws.Web.csproj

```diff
@@ -1225,6 +1230,7 @@
     <Compile Include="Areas\AdminExportAssessment\Controllers\ProducerController.cs" />
     <Compile Include="Areas\AdminExportAssessment\Controllers\UpdateJourneyController.cs" />
     <Compile Include="Areas\AdminExportAssessment\ViewModels\Annex\DownloadAnnexViewModel.cs" />
+    <Compile Include="Areas\AdminExportAssessment\ViewModels\Comments\AddCommentsViewModel.cs" />
     <Compile Include="Areas\AdminExportAssessment\ViewModels\Comments\CommentsViewModel.cs" />
```

Comment:
Build is currently failing because it's missing entry for the IMPORT `AddCommentsViewModel`. Think it's been removed as one of the merge.